boa/environment.py

```
@@ -399,14 +399,13 @@ def set_random_seed(self, seed=None):
     def get_gas_price(self):
         return self._gas_price or 0
 
-    def _set_account_db_class(self, account_db_class: type):
-        self.vm.__class__._state_class.account_db_class = account_db_class
-
     def _init_vm(self, reset_traces=True):
         self.vm = self.chain.get_vm()
-
         self.vm.patch = VMPatcher(self.vm)
 
+        # revert any previous AccountDBFork patching
+        self._set_account_db_class(AccountDB)
```

this change got lost and broke anvil, so the error did not need to be swallowed